lms/djangoapps/teams/models.py

```diff
@@ -223,7 +228,7 @@ def __setattr__(self, name, value):
                     )
         super(CourseTeamMembership, self).__setattr__(name, value)
 
-    def save(self, *args, **kwargs):
+    def save(self, *args, **kwargs): # pylint: disable=arguments-differ
```

Comment:
The `pylint: disable` is good, but the style checker will yell at you for having one space instead of two in front of the `#`.

You can run the style checker locally with `pycodestyle lms/djangoapps/teams`.

(The difference between `pycodestyle` and `pylint` is that the former catches formatting issues, where as the latter catches potential logical issues).